fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -74,6 +76,66 @@ public static void checkEventNameUniqueness(@Nonnull Stream<Event> events) {
         }
     }
 
+    /**
+     * Subtracts counts and times recorded by a snapshot of a timer.
+     *
+     * Subtracting a snapshot of a timer from the original timer after subsequent operations have
```

Comment:
If you want these to appear as paragraphs in the published docs, then you should have a `<p>` at the beginning of each paragraph.

Reply:
Doh! The code formatting tool kept putting them in and I kept taking them out because I didn't see them in other parts of the code. Will leave them in even though they remind me of the nightmare of XML.